examples/hello_workflow/workflow.yaml

```diff
@@ -1,5 +1,5 @@
+apiVersion: "dag.example.com/v1"
```

Comment:
Shouldn't this be something other than "example.com".  I don't know... dag.acs.amadeus.com?

Reply:
Right

Reply:
It's going to be something configurable. But I'm going to do in another PR. Thanks